test/XRP/fakes/fake-xrp-client.ts

```diff
@@ -67,6 +67,14 @@ class FakeXrpClient implements XrpClientDecorator {
     return FakeXrpClient.returnOrThrow(this.getPaymentValue)
   }
 
+  async enableDepositAuth(_wallet: Wallet): Promise<string> {
+    // NOTE: all we need to do here is return a valid transaction hash, which we already have in the field this.sendValue
```

Comment:
I'd go with the latter - a generic transaction hash. 

If we ever need something more advanced we can always easily refactor this API with breaking changes since it is only test code.